src/focus-trap-react.js

```diff
@@ -45,6 +66,10 @@ class FocusTrap extends React.Component {
     if (this.props.paused) {
       this.focusTrap.pause();
     }
+
+    // if there is a _FocusTrapReact context from a parent focus trap, pause it
+    const {_FocusTrapReact: {pause} = {}} = this.context;
```

Comment:
I'd prefer some more straightforward non-destructured code here — something like `if (this.context.FocusTrapParent && this.context.FocusTrapParent.pause) { .. }` — especially because this instance has its own `pause` function.